pkg/controller/user/saveuser.go

```diff
@@ -47,38 +46,26 @@ func NewSaveController(ctx context.Context, config *config.ServerConfig, db *dat
 }
 
 func (usc *userSaveController) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	defer http.Redirect(w, r, "/users", http.StatusSeeOther)
-
-	user, err := controller.GetUser(w, r)
-	if err != nil {
-		http.Redirect(w, r, "/signout", http.StatusFound)
-		return
-	}
 	flash := flash.FromContext(w, r)
 
-	usc.logger.Infof("FLASH: %+v", flash)
-
-	m := html.GetTemplateMap(r)
-	m["user"] = user
-
 	var form formData
 	if err := controller.BindForm(w, r, &form); err != nil {
-		usc.logger.Errorf("error parsing form: %v", err)
 		flash.Error("Failed to process form: %v", err)
-		controller.WriteJSON(w, http.StatusBadRequest, nil)
```

Comment:
Unclear why we were returning JSON?

Reply:
yeah - redirect is right here, it's a human used page.